force_wfmanager/central_pane/setup_pane.py

```diff
@@ -31,3 +124,71 @@ def _console_ns_default(self):
             namespace["app"] = None
 
         return namespace
+
+    # Properties
+
```

Comment:
If and object does not have a 'custom' view, we don't want to show the default one as there are a lot of traits which are not intended to be user-facing. I imagine there may well be a cleaner way to check for this though